### DISCUSSION TITLE
Search result - selected properties

### DISCUSSION THREAD

**Discussion by matjazreberc** [microsoft-search/pnp-modern-search#3081] [Q&A]
Hello,
I need help understanding what are "Selected properties" in Pnp Search Result web part.
Arent this "Selected properties" columns which will be searched and only data returned if something maches in my selected properties?

My configuration:

![image](https://github.com/microsoft-search/pnp-modern-search/assets/60425382/a2770b55-d58e-40dd-9e5c-eae82bf7b494)

but the data returned are also with RefinableString**: null

![image](https://github.com/microsoft-search/pnp-modern-search/assets/60425382/5ca095af-f946-496b-a7ce-ed238ab50187)

Am I doing something wrong are I don't understand how it works?

Thank you, Matjaz

**Comment by wobba** [comment]
Selected properties is the list of properties you want to be returned per item in the result. Values you want to use in the UX somehow when displaying the result. It has nothing to do with what you are searching on. And they have to be `retrievable` properties in the search schema.
